packages/@okta/vuepress-site/docs/guides/device-context/main/index.md

```diff
@@ -52,7 +52,7 @@ Update your Okta org to allow for AMFA. The following steps set up your org to p
 
 1. If you haven't yet created an org, [create one](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#get-set-up) and <StackSnippet snippet="configureorg" inline/>.
 
-1. Ensure that you have at least one authenticator set up for your org. See how to set up an [Email](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#_1-set-up-the-email-authenticator-for-authentication-and-recovery) and [Phone](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#_2-add-the-phone-authenticator-for-authentication-and-recovery) aAthenticator.
+1. Ensure that you have at least one authenticator set up for your org. See how to set up an [Email](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#_1-set-up-the-email-authenticator-for-authentication-and-recovery) and [Phone](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#_2-add-the-phone-authenticator-for-authentication-and-recovery) Authenticator.
```

Comment:
> "Ensure that you've enabled at least one authenticator in your org."

Reply:
"Ensure that at least one authenticator is enabled in/for (configured in/for) your org."